File: src/containsDuplicate.java
import java.util.Arrays;

public class containsDuplicate {
    public static void main(String[] args) {
        System.out.println(containsDuplicate(new int[] {0, 1, 2, 0}));
    }

    public static boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for (int i = 0; i < nums.length-1; i++) {
            for (int j = i+1; j < nums.length; j++) {
                if (nums[i] == nums[j]) return true;
            }
        }
        return false;
    }

}

File: arrays_and_hashing/contains_duplicate.py
from typing import List

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return len(set(nums)) != len(nums)
    
if __name__ == "__main__":
    solution = Solution()
    print(solution.containsDuplicate([1, 2, 3, 1]))
    print(solution.containsDuplicate([4, 2, 3, 1]))
